packages/canon/report.api.md

```diff
@@ -214,6 +289,49 @@ export type FlexDirection = 'row' | 'column';
 // @public (undocumented)
 export type FlexWrap = 'wrap' | 'nowrap' | 'wrap-reverse';
 
+// @public (undocumented)
+export const gapPropDefs: {
+  gap: {
+    type: 'enum | string';
+    className: string;
+    customProperties: '--gap'[];
```

Comment:
Do these need to be part of the public API?

Reply:
No you're right. I'll refactor that in a following PR. I was mostly trying to get the warnings out but I need to clean this up a little.